<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(162, 109, 159);</string>
    </property>
    <widget class="QPushButton" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>290</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Statut :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>mode de paiement :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date_Commande :</string>
     </property>
    </widget>
    <widget class="QLabel" name="le_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Id_commande :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>371</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>COMMANDES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chercher">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>50</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
    </widget>
    <widget class="QLabel" name="PREN">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> ID :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trierbutton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>110</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_paiement">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_statut">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>270</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_date">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimerbutton">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>400</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rechercher_id">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="modifierbutton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>250</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>190</y>
       <width>331</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 115, 163);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="supp">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>400</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="le_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>400</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Afficher">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>210</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>Afficher </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonASC">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>110</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ascendant</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDESC">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>110</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Descendant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdfButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>249</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficherStatistiques">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>249</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);</string>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkboxListe">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>231</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Exporter liste des commandes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Quitter">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>10</y>
       <width>71</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 119, 178);
</string>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_stat">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>470</y>
       <width>671</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Pictures/Screenshots/fash.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
